### PR TITLE
Add rdb-recording service

### PR DESCRIPTION
**Pull request by marcingesek79**

### CODE REVIEW

**Review by dideek — APPROVED** [review]

File: grisera_api/recording/recording_service_relational.py [hunk 1]
```diff
@@ -1,5 +1,120 @@
...
 class RecordingServiceRelational(RecordingService):
-    pass
+    
+    def __init__(self):
+        self.rdb_api_service = RdbApiService()
+        self.table_name = Collections.RECORDING
+
+
+    def save_recording(self, recording: RecordingIn):
+        recording_data = {
```
Comment (Noxxxxious) [inline grisera_api/recording/recording_service_relational.py:17]: Missing participation_id and registered_channel_id

File: grisera_api/recording/recording_service_relational.py [hunk 2]
```diff
@@ -1,5 +1,120 @@
...
+            "additional_properties": json.dumps([
+                {
+                    "key": p.key,
+                    "value": p.value
+                } for p in recording.additional_properties
+            ])
+        }
+        result = self.get_recording(recording_id)
+        if type(result) != NotFoundByIdModel:
```
Comment (Noxxxxious) [inline grisera_api/recording/recording_service_relational.py:79]: I think using a guard clause: type(result) == NotFoundByIdModel: return ... is more readable, this however doesn't  change how the service works so it's optional.

File: grisera_api/recording/recording_service_relational.py [hunk 3]
```diff
@@ -1,5 +1,120 @@
...
+            if put_result["errors"] is not None:
+                    return RecordingOut(errors = put_result["errors"])
+            return RecordingOut(**put_result["records"])
+        return result
+    
+
+    def update_recording_relationships(self, recording_id: Union[int, str],
+                                       recording: RecordingRelationIn):
+        result = self.get_recording(recording_id)
+        if type(result) != NotFoundByIdModel:
```
Comment (Noxxxxious) [inline grisera_api/recording/recording_service_relational.py:90]: Same as above

File: grisera_api/recording/recording_service_relational.py [hunk 4]
```diff
@@ -1,5 +1,120 @@
...
+        return RecordingOut(**result["records"])
+
+
+    def get_recordings(self):
+        results = self.rdb_api_service.get(self.table_name)
+        return RecordingsOut(recordings=results)
+
+
+    def get_recording(self, recording_id: Union[int, str], depth: int = 0, source: str = ""):
+        import registered_channel.registered_channel_service_relational as rc_rel
```
Comment (Noxxxxious) [inline grisera_api/recording/recording_service_relational.py:39]: Wouldn't import, import import, service, service, service look better?

**Review by Noxxxxious — APPROVED** [review]

**Pull request merged**